.storybook/pages/CadStep1/CadStep1.module.css

```diff
@@ -0,0 +1,70 @@
+/*------------------------------------*\
+    # CAD Step 1
+\*------------------------------------*/
+
+/**
+ * Page Layout
```

Comment:
I think something overall that we try to do is limit any sort of page-specific CSS for the most part so that users of the system don't have to think much about that. One thing that I've been curious about from a utility class standpoint is not making those `Spacing.module.css` so that users could apply utility classes to space items out if need be. That's generally how we tend to build out pages is with a lot of margin bottom utilities to make it easier on consuming developers. From a design standpoint, there is usually a convention if there needs to be responsive behavior in between items (e.g. we've heard this called `major-section-break`), we can create a utility class around that so that users building out pages can use that between those sections to get `2rem` of spacing between sections on small screens and then `4rem` or spacing on large screens. I'd be curious as to what @dierat @anniehu4  @csnizik think of this or if we should actually have a discussion about this. Thanks!

Reply:
yep, I totally agree with pulling utilities out of CSS modules! I think these got mass converted all at the beginning 🙂

Reply:
Should I convert all the stylesheets in `src/components/Utilties` into regular CSS files and change the consuming files appropriately?

Reply:
And as for responsive spacing, it would be unideal to create utility classes for various permutations of spacings, how is that usually decided?